smart-contracts/contracts/RelayPool.sol

```diff
@@ -141,8 +140,6 @@ contract RelayPool is ERC4626, Ownable {
     address wrappedEth,
     address curator
   ) ERC4626(asset, name, symbol) Ownable(msg.sender) {
-    // TODO: can we verify that the asset is an ERC20?
```

Comment:
This would be user error and no need to handle here (would add extra gas for little/no benefit). Also, soon the first deposit will happen in factory anyway,... so that would fail if not an ERC20